README.md

```diff
@@ -44,6 +44,13 @@ can be replaced with
 async def async_app(): ...
 ```
 
+It's also possible to run a main function from another module:
```

Comment:
Since this PR effectively adds support for tools like `typer` and `click`, it might be cool to include a short example of that here as well.